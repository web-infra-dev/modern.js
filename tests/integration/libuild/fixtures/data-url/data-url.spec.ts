import { getLibuilderTest } from '@/toolkit';

describe('data-url', () => {
  it('import', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
    });
    await bundler.build();
    bundler.expectJSOutputMatchSnapshot();
  });
});
