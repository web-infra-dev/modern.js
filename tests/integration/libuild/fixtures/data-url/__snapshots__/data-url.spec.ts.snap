// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`data-url import 1`] = `
"// <data:text/javascript,export default 42>
var javascript_export_default_42_default = 42;

// index.ts
console.log(\\"x:\\", javascript_export_default_42_default);
"
`;
